source/include/common/cs_Types.h

```diff
@@ -471,7 +477,7 @@ typedef uint32_t TYPIFY(STATE_BEHAVIOUR_MASTER_HASH);
 typedef cs_mesh_iv_index_t TYPIFY(STATE_MESH_IV_INDEX);
 typedef cs_mesh_seq_number_t TYPIFY(STATE_MESH_SEQ_NUMBER);
 typedef ibeacon_config_id_packet_t TYPIFY(STATE_IBEACON_CONFIG_ID);
-typedef cs_microapp_t TYPIFY(STATE_MICROAPP);
+typedef microapp_state_t TYPIFY(STATE_MICROAPP);
```

Comment:
include cs_microapppackets.h, currently indirectly included through #include cs_packetsinternal.h which then does #include cs_packets.h to finally get to #incude said header

Reply:
That's how we do it for all.

Reply:
I'm including the 'include what you use' paradigm to STYLE.md. cs_Types.h will be better off adhering to it at some point. I'll let it be for this PR. :-)